src/main/java/com/iopipe/IOpipeExecution.java

```diff
@@ -587,16 +612,20 @@ final RemoteRequest __buildRequest()
 				
 				CustomMetric metric = custmetrics[i];
 				
-				gen.writeStartObject();
-				
-				gen.write("name", metric.name());
-				
-				if (metric.hasString())
-					gen.write("s", metric.stringValue());
-				if (metric.hasLong())
-					gen.write("n", metric.longValue());
-				
-				gen.writeEnd();
+				String xname = metric.name();
+				if (IOpipeExecution.__isNameInLimit(xname))
+				{
+					gen.writeStartObject();
+					
+					gen.write("name", xname);
+					
+					if (metric.hasString())
+						gen.write("s", metric.stringValue());
+					if (metric.hasLong())
+						gen.write("n", metric.longValue());
+					
+					gen.writeEnd();
+				}
```

Comment:
If it's not in the name limit, can we print a (debug or warn) log message for users?

Reply:
Did this!